src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -26,6 +27,34 @@ const timecardService = require('../../../api/services/timecardService');
 const mockCreateTimeEntry = jest.spyOn(timecardService, 'createTimeEntry');
 const mockUpdateTimeEntry = jest.spyOn(timecardService, 'updateTimeEntry');
 
+const invalidTimes = [
+  '-00:01',
+  '24:00',
+  'abcd',
+  '!',
+  '25',
+  '13am',
+  '24pm',
+  '3p',
+  '6m',
+  '7a',
+];
+
+const validTimes = [
```

Comment:
can we add an AM double digit time here. 
e.g. 10

Reply:
As above, I've removed attempting to handle am/pm in the input box